css/style.css

```diff
@@ -0,0 +1,1138 @@
+  flex-direction: row-reverse;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.row__test {
+  display: flex;
+}
+
+.main__img-steering {
+  float: left;
+  margin-top: 20px;
+  margin-right: 8px;
+}
+
+.main__img-steering-1,
+.main__img-steering-2 {
+  display: none;
+}
+
+.main__row-text {
+  display: flex;
+  flex-wrap: row nowrap;
+  align-items: center;
+  font-weight: bold;
+  text-align: center;
+  font-size: 13px;
+  color: black;
+  margin-top: 23px;
+}
+
+.button {
+  margin: 0;
+  display: flex;
+  align-items: center;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 13px;
+  text-align: center;
+  border: 2px solid #e51937;
+  width: 160px;
+  height: 40px;
+  padding: 10px 20px;
+}
+
+.main__button {
+  position: relative;
+  color: #e51937;
+  background-color: white;
+  text-align: center;
+  justify-content: center;
+  margin-left: 52px;
+  margin-top: 11px;
+  right: 10px;
+}
+
+.main__button:hover {
+  box-shadow: 0 5px 8px #3c354a;
+  transition: 0.3s all ease-out;
+  cursor: pointer;
+}
+
+.main__arrow {
+  display: flex;
+  justify-content: center;
+  margin-top: 25px;
+}
+
+@media (max-width: 1270px) {
+  .header {
+    padding: 24px 33px 0 15.32px;
+  }
+
+  .header__img-burger {
+    width: 35.23px;
+  }
+
+  .main {
+    background: url(../img/Tesla_logo_car_ipad.png) no-repeat;
+    background-size: contain;
+    margin-top: 178px;
+    padding-top: 1px;
+  }
+
+  .main__title {
+    margin-top: -99.86px;
+    margin-bottom: 5px;
+    justify-content: center;
+  }
+
+  .main__description {
+    justify-content: center;
+    font-size: 15px;
+    margin-top: -12px;
+    text-align: center;
+    margin-left: 0;
+  }
+
+  .main__container {
+    flex-direction: column;
+    padding: 0 140px;
+    margin-top: 725.49px;
+  }
+
+  .main__container-second {
+    bottom: 28px;
+  }
+
+  .main__row {
+    margin-bottom: 57px;
+    margin-left: 47px;
+    margin-right: 90px;
+  }
+
+  .main__row-title {
+    font-size: 50px;
+    margin: -6px 19px -4px 0;
+  }
+
+  .main__row-test {
+    margin-left: -42px;
+  }
+
+  .row__test {
+    display: flex;
+    margin-top: 32px;
+  }
+
+  .main__row-text {
+    color: #e51937;
+    margin-top: 0;
+    text-shadow: none;
+    font-size: 16px;
+    margin-left: 15px;
+  }
+
+  .button {
+    margin: 0;
+    justify-content: center;
+    background-color: #e51937;
+    color: white;
+  }
+
+  .main__button {
+    width: 220.84px;
+    height: 57.13px;
+    left: 0;
+    margin: 17px 0 0 39px;
+  }
+
+  .main__img-vector {
+    width: 75.92px;
+    padding-left: 0;
+  }
+
+  .main__img-stroke {
+    width: 50.83px;
+    margin-right: 0;
+  }
+
+  .main__img-steering,
+  .main__img-steering-2 {
+    display: none;
+  }
+
+  .main__img-steering-1 {
+    display: inline-block;
+    margin-left: 48px;
+  }
+
+  .main__subtitle {
+    font-size: 15px;
+  }
+
+  .main__row-desc {
+    font-size: 12px;
+    text-align: left;
+    margin-top: 14px;
+    right: -13px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-Project_Evgen/pullRequest?prid=6071575)